react/src/util/opReturn.ts

```diff
@@ -1,20 +1,69 @@
-export const OP_RETURN_PREFIX = 'paybutton$';
+import { lib, enc } from 'crypto-js';
 
-export function stringToHex(str: string): string {
+//opReturn format is:
+//`04` - Pushdata of the Protocol Identifier, `50415900`
+//`50415900` - PayButton protocol identifier, ascii "PAY"
+//`00` - Version 0, pushed as `OP_0`
+//`09` - pushdata for the data payload, signifying this tx has 9 bytes of data
+//`0102030405060708aa` - data payload
+//`08` - pushdata for the optional nonce payload, signifying this tx has 8 bytes of nonce data
+//`0102030405060708` - The 8-byte nonce
+
+// All the below variables are already encoded to HEX
+export const OP_RETURN_PREFIX_PUSH_DATA = '04'; // \x04
+export const OP_RETURN_PREFIX = '50415900'; // PAY\x00
+export const VERSION = '00'; // \x00
+
+// 220 total bytes
```

Comment:
223 total bytes

1 for OP_RETURN `6a`
1 for 04 lokad id pushdata
4 for lokad id
1 for OP_0 version byte
...
per draft spec, the 8-byte nonce is optional and not always present? So sometimes these bytes are available for a data push?

1 or 2 pushdata bytes for the data push depending on size of data

...remainder for data

Reply:
- Fixed the math; thanks
- Also did the division to better clarify what is being subtracted.
- As for the 'optionality' of the 8-byte nonce, indeed I'll implement this in another PR (it will require a new set of props, checks, etc so I prefer to do it separately)
- pushData will never be more than one byte because the data will never be bigger than 255 bytes; since the script it self is limited to less than that.